src/locale/i18n.ts

```diff
@@ -2,12 +2,14 @@ import en from "locale/en";
 import { merge } from "lodash";
 import { createIntl, createIntlCache } from "react-intl";
 
-import { apiClient } from "../utils/ApplicationContext";
+// import { apiClient } from "utils/ApplicationContext";
 
 async function loadLocaleData(locale: string): Promise<Record<string, string>> {
     let backend_messages;
     try {
-        backend_messages = await apiClient.translations(locale);
+        backend_messages = {};
+        // circular import
+        // backend_messages = await apiClient.translations(locale);
```

Comment:
What causes the circular import? I don't see which module is referring back to i18n.ts

Anyway maybe you can import it inside the function to prevent it, I don't know the exact syntax but should be something like `let apiClient = require("utils/ApplicationContext").default;`

Reply:
WIP. I used a tool for checking the circulars. This will be fixed soon. :)

Reply:
Probably the other circular imports had a negative effect on this: it now starts without the error...